spec/unit/bootstrap/definitions_spec.rb

```diff
@@ -0,0 +1,29 @@
+describe FHIR::Definitions do
+  context '.load_extensions'
```

Comment:
Do you mean that `FHIR::Definitions#load_extensions` isn't tested here, or that `get_display` isn't?